index.bs

```diff
@@ -2645,6 +2812,9 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
         1. [=set/Append=] |rateLimitRecord| to the [=attribution rate-limit cache=].
 1. Run [=obtain and deliver a debug report on source registration=] with |debugDataType| and |source|.
 1. [=set/Append=] |source| to |cache|.
+1. [=Remove sources with different max event states if necessary=] with |source|.
```

Comment:
could you clarify `when should the check be performed`? 

I don't think verbose debug report support has been fully scoped yet. Not sure if we want to address it in this version :)

Reply:
Basically I mean the order of the checks in the source registration. For example, whether it should be checked before step 17 of https://wicg.github.io/attribution-reporting-api/#process-an-attribution-source, i.e. before any limits that are not reported explicitly.

Even if verbose debug report is not supported yet, I think we should address this now, as changing the order would be backwards incompatible strictly speaking and I hope to avoid.

Reply:
`Remove sources with different max event states if necessary` (the 2 after this) will only affect older sources. The current source has been fully validated and should be stored if it reaches step 17.

Reply:
Moved the section before security checks.